src/components/shapes/draw_newshape/attributes.js

```diff
@@ -78,7 +79,85 @@ module.exports = {
                 '*vertical* allows vertical extend.'
             ].join(' ')
         },
-
+        label: {
+            text: {
+                valType: 'string',
+                dflt: '',
+                editType: 'none',
+                description: 'Sets the text to display with shape.'
```

Comment:
All the descriptions in this file should mention `'new' shape` instead of `shape`.